include/swift/ClangImporter/ClangImporter.h

```diff
@@ -78,14 +80,20 @@ class ClangImporter final : public ClangModuleLoader {
   /// \param ctx The ASTContext into which the module will be imported.
   /// The ASTContext's SearchPathOptions will be used for the Clang importer.
   ///
-  /// \param clangImporterOpts The options to use for the Clang importer.
+  /// \param importerOpts The options to use for the Clang importer.
+  ///
+  /// \param swiftPCHHash A hash of Swift's various options in a compiler
+  /// invocation, used to create a unique Bridging PCH if requested.
+  /// \see -pch-output-dir
   ///
   /// \param tracker The object tracking files this compilation depends on.
   ///
   /// \returns a new Clang module importer, or null (with a diagnostic) if
   /// an error occurred.
   static std::unique_ptr<ClangImporter>
-  create(ASTContext &ctx, const ClangImporterOptions &clangImporterOpts,
+  create(ASTContext &ctx,
+         ClangImporterOptions &importerOpts,
+         std::string swiftPCHHash,
```

Comment:
All right, that's fair. Do you know if there is any place that I can ask for this state just by having already imported it or do we need to add a bit to `ClangImporter::Implementation` or something?